projects/kit/components/input-slider/test/input-slider.component.spec.ts

```diff
@@ -354,101 +354,6 @@ describe(`InputSlider[new]`, () => {
         });
     });
 
-    describe(`\`min\` prop`, () => {
```

Comment:
All this deleted `describe` is replaced by Cypress tests.

See `projects/demo-integrations/cypress/tests/kit/input-number/input-number.cy.ts`